### ISSUE TITLE
QR code not recognised when using 'Choose file' option - when hosting site doesn't include a border around the QR code

### ISSUE THREAD

**Issue by tficharmers** [mymonero/mymonero-app-js#267]
**Not really a MyMonero bug, but mentioning it anyway.**

Versions affected: MacOS 1.1.4 & iOS 1.1.4

I tried to donate to VLC using the iOS app, whilst not leaving my phone.

https://www.videolan.org/contribute.html

I went to the above page and saved the QR image to Photos. I then went into MyMonero and pressed 'Choose file', selected the QR code, but was left with an error. I then tried it on the MacOS app and got a similar error:

iOS: "Unable to find QR code data in image"
MacOS: "MyMonero was unable to find a QR code in that image"

Interestingly, using the iOS app to capture the QR code off a monitor using the camera, does work. I think this is because the VLC web team hasn't saved their images with a white border. I doubt it would be MyMonero's job to do add this when it is uploading them to check. After doing some tests though, it appears all it needs is a 1px white border adding for MyMonero to identify it.

**Comment by paulshapiro** [comment]
It's interesting that a border fixes it. Does the border fix it on both platforms?

**Comment by tficharmers** [comment]
Yes. The 1px white border version works on iOS too. This was a problem in the official GUI too:

https://github.com/monero-project/monero-gui/issues/1557

Looking at the VLC QR images, they have saved both the Monero and Bitcoin QR codes with transparent borders, which of course gets ignored. White or light grey is required it seems

**Comment by paulshapiro** [comment]
Interesting. This'll be easy to solve under iOS, but probably slightly more involved in JS. Thanks for the report!
